src/engine/Actions/Actions.stories.ts

```diff
@@ -0,0 +1,31 @@
+import { storiesOf } from '@storybook/html';
+import { withKnobs, number } from '@storybook/addon-knobs';
+import { Actor, Texture, Loader } from '../';
+import { withEngine } from '../story-utils';
+
+import heartTexture from '../__assets__/heart.png';
+
+storiesOf('Actions', module)
```

Comment:
This story syntax will get simpler when I install the Storybook beta package. It will change to

```ts
export default {
  title: "Actions",
  decorators: [withKnobs]
}

export const fade = withEngine(game => { ... });
fade.story = {
  parameters: {
    notes: '...'
  }
}
```